build-bin/docker-compose.test.yml

```diff
@@ -7,7 +7,7 @@ services:
     image: ghcr.io/openzipkin/alpine:3.12.1
     entrypoint: /bin/sh
     # Keep the container running until HEALTHCHECK passes
-    command: "-c \"sleep infinity\""
```

Comment:
infinity isn't always supported in versions of sleep,  I suppose you have to wake sometime